### PR TITLE
feat(us): activate veeam backup

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: constants.config.js [hunk 1]
```diff
@@ -826,7 +826,8 @@ var constants = {
                 faqVps: "https://www.ovh.com/ca/en/vps/faq-help.xml",
                 dedicatedOrder: "https://us.ovhcloud.com/order/dedicated/#/dedicated/select",
                 vrackOrder: "https://us.ovhcloud.com/order/express/#/express/review?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
-                cloudProjectOrder : "https://us.ovhcloud.com/manager/cloud/#/iaas/pci/offer"
+                cloudProjectOrder : "https://us.ovhcloud.com/manager/cloud/#/iaas/pci/offer",
```
Comment (antleblanc) [inline constants.config.js]: `cloudProjectOrder :` > `cloudProjectOrder:`

File: client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    </select>
+                    <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
+                </label>
+                <div class="oui-field__helper mt-3"
+                     data-translate="dedicatedCloud_datacenter_backup_enable_version_explain"
+                     data-translate-values="{ t0: $ctrl.veamBackupUrl }">
+                </div>
+            </oui-field>
+        </div>
+
```
Comment (antleblanc) [inline client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.html]: You can remove empty lines:)

File: client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+               data-primary-label="{{ ($ctrl.hosts.length && $ctrl.offer ? 'dedicatedCloud_options_activate' : 'global_OK') | translate }}"
+               data-secondary-label="{{ $ctrl.hosts.length && $ctrl.offer ? ('wizard_cancel' | translate) : '' }}"
+               data-secondary-action="$ctrl.closeModal()"
+               data-on-dismiss="$ctrl.closeModal()"
+               data-loading="$ctrl.loading.init"
+               data-type="{{ !$ctrl.loading.init && (!$ctrl.hosts.length || !$ctrl.offer) ? 'warning' : ''}}">
+
+        <p data-ng-if="(!$ctrl.hosts.length || !$ctrl.offer) && !$ctrl.loading.init"
+           data-translate="{{ !$ctrl.hosts.length ? 'dedicatedCloud_tab_veeam_on_disabled' : 'dedicatedCloud_datacenter_backup_enable_no_offer'  }}"
+           data-translate-values="{ t0: $ctrl.datacenter.name }">
```
Comment (antleblanc) [inline client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.html]: Does `t0` could be `name` for instance?

File: client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.controller.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      }],
+    };
+
+    return this.User.getUrlOf('express_order')
+      .then((url) => {
+        this.$window.open(`${url}review?products=${JSURL.stringify([productToOrder])}`, '_blank');
+      })
+      .catch((error) => {
+        this.Alerter.error([this.$translate.instant('dedicatedCloud_tab_veeam_enable_fail'), _.get(error, 'data.message')].join(' '));
+        this.closeModal();
```
Comment (marie-j) [inline client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.controller.js]: Not necessary if you call it in `finally`

Comment (Jisay) [reply client/app/dedicatedCloud/datacenter/backup/enableAgora/dedicatedCloud-datacenter-backup-enableAgora.controller.js]: indeed :-)

**Pull request merged**